src/services/listS3Objects.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  S3Client,
+  ListObjectsV2Command,
+  ListObjectsV2CommandInput
+} from '@aws-sdk/client-s3'
+
+export const listS3Objects = async (
+  input: ListObjectsV2CommandInput,
+  objects: string[] = []
+): Promise<string[]> => {
+  const client = new S3Client({ region: 'eu-west-2' })
+  const command = new ListObjectsV2Command(input)
+  const response = await client.send(command)
+
+  if (response.Contents) {
```

Comment:
We could reduce nested if statements here if we return the object when there's no content.